docs/meetings/client/2022_12_02.md

```diff
@@ -0,0 +1,59 @@
+# Client Meeting Minutes
+
+- **Start Time:** 4:00 PM
+- **End Time:** 4:25 PM
+- **Location:** Zoom
+- **Date:** December 2, 2022
+
+## Attendance
+
+- Client:
+  - Nazim Ragimov (CEO)
+  - Erik (dev)
+- Team A:
+  - Veronica Jack
+  - Edouard Eltherington (laptop died; on mobile, audio only)
+  - Logan Parker
+  - Matthew Kuelker
+
+### Attendance Comments
+
+- Other teams present (unclear if all team members were present)
+  - Team B
+  - Team C
+
+## Team A
+
+- Quick overview of the week:
+  - we have great feedback from our peer testing sessions so we have some stuff to improve
+
+- our goals for next semester will be animating the face and body, along with improved realism
+- our question about making the program a package or something instead of API
+  - Erik: whatever works for us, as long as it can integrate with their system
+  - Logan: sending over HTTP, it might be slower with the audio and text files, so to avoid additional time sending...
+  - will stick with original plan of API
+- determine when we'll message them in January to determine the new meeting schedule
+
+## Team B
+
+- this week, they were working on their animator request flushed out
+- will send the word document outlining what frames are needed (mouths and poses), animator notes today
+- 22 different mouths, many may overlap with other teams
+- same characters, or easy colour changes
+- API is almost done, needs to be uploaded
+
+## Team C
+
+- does Kukarella have a virtual private server to test on?
+  - Erik: maybe? ask Group B what they did or try a cloud service for free trial, or just have local API working. It
+would be more difficult to add us to theirs, and it's not necessary for them to host it until they receive our stuff.
+- The API for the demo API on the front page, is that fine for us to use?
+  - Erik: the demo is fine to use, they charge ~10-15 for 1 million characters
+- The free API demo... we can make 20-30 calls within a couple minutes - just wanted to check on blocks
+  - Erik: thank you for pointing that out, it was made a while ago so we'll check it out
+
+## To-do List
+
+- Animator may need feedback over the break, so please have at least 1 person per team available on Discord to check
+every couple days just in case
+- Determine scheduling for next semester for the client meeting

```

Comment:
Jan 8. check in?

Reply:
Jan 8 sounds good! I'll post in Discord about this, too!